### DISCUSSION TITLE
This version of ChromeDriver only supports Chrome version 111

### DISCUSSION THREAD

**Discussion by rafaelfliu** [ultrafunkamsterdam/undetected-chromedriver#1112] [Q&A]
I encountered an error while running undetected_chromedriver with the headless option:

selenium.common.exceptions.WebDriverException: Message: unknown error: cannot connect to chrome at 127.0.0.1:53933
from session not created: This version of ChromeDriver only supports Chrome version 111
Current browser version is 110.0.5481.100

chrome: 110.0.5481.100
selenium: 4.8.2
undetected_chromedriver: 3.4.6

i wonder if someone had encountered the same issue?

**Comment by rafaelfliu** [comment]
not sure why i did not see this just now, but the solution is this:

```
   import undetected_chromedriver as uc
   driver = uc.Chrome( version_main = 110 )
```

**Reply by AndreFK** [reply]
We ran into this same issue today and yeah that was the solution. I'm not sure why we got that error just now though, we haven't even updated anything so idk what was it that caused it.

**Reply by idomanteu** [reply]
Hey, I'm receiving this error when I try that fix. 
TypeError: __init__() got an unexpected keyword argument 'version_main'

**Reply by rafaelfliu** [reply]
my guess is, you entered this in ChromeOptions(), instead of of Chrome()?

**Comment by huyangnl** [comment]
You should upgrade your chome to 111 or you have to set version_main = 110 in your chrome option.
